### ISSUE TITLE
OverflowSet: Vertical is using role="menubar"

### ISSUE THREAD

**Issue by micahgodbolt** [OfficeDev/office-ui-fabric-react#11389]
Vertical overflow sets are using default menubar role, which is specifically for horizontal menus. Role menu should be used instead.
